settings/config.settings.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+// Ensure the appropriate config split is enabled.
+$config['config_split.config_split.local']['status'] = FALSE;
+$config['config_split.config_split.dev']['status'] = FALSE;
+$config['config_split.config_split.stage']['status'] = FALSE;
+$config['config_split.config_split.prod']['status'] = FALSE;
+$config['config_split.config_split.ci']['status'] = FALSE;
```

Comment:
I'm also using `test` (mainly because that's what canary does... I haven't tested to see how it works yet).

Besides RA, a project could have any number of custom environments (typically `dev1`, `dev2`, etc... but they could be named anything). I think it would be awesome if we could create this configuration automatically, or as part of a wizard (i.e. "here are the typical environments, do you have any others?"). Short of that, just documenting it would be good.